### ISSUE TITLE
how to fine-tune qwen-image-layered

### ISSUE THREAD

**Issue by garychan22** [QwenLM/Qwen-Image-Layered#27]
Hi, thanks for the excellent model here.

I am trying to fine-tune this model by adapting the diffusers script of Qwen-Image from https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth_lora_qwen_image.py with minimal modifications:

1. replace corresponding components and pipeline for layered model
2. feed RGBA image data:  target is [image, layer0, layer1, layer2], input is image

However, the model didn't converge (loss around 1.00+) for 200 samples with batch size 32 and 40K iterations in total. Do I need to change the noise scheduler or the loss computation, though I have checked the details from the paper and found that it is the similar loss to Qwen-Image?

Thanks for help!

**Comment by garychan22** [comment]
possibly already fixed the training issue with diffusers scripts, loss less than 0.1 and the inference outputs became better
